calour/io.py

```diff
@@ -548,7 +512,7 @@ def read_amplicon(data_file, sample_metadata_file=None,
     if min_reads is not None:
         exp.filter_by_data('abundance', axis=0, cutoff=min_reads, mean_or_sum='sum', inplace=True)
     if normalize is not None:
-        exp.normalize(total=normalize, axis='s', inplace=True)
+        exp.normalize(total=normalize, axis=0, inplace=True)
```

Comment:
i think axis='s' is more understandable to someone reading the code compared to axis=0